daemon/src/projection.rs

```diff
@@ -1371,20 +1368,6 @@ impl CfdOffer {
             .leverage_choices
             .iter()
             .map(|leverage| {
-                let liquidation_price = match own_position {
```

Comment:
🤔 This removes information from the http API, but only the taker user interface was adapted - the maker UI is not using these values, or we just don't care about updating the deprecated UI?

Will this require changes for other consumers of the API?